src/sentry/consumers/__init__.py

```diff
@@ -61,6 +61,20 @@ def multiprocessing_options(
     ]
 
 
+def issue_occurrence_options() -> list[click.Option]:
+    """Return a list of issue-occurrence options."""
+    options = multiprocessing_options(default_max_batch_size=100)
+    options.append(
+        click.Option(
+            ["--mode", "mode"],
+            type=click.Choice(["batched-parallel", "parallel"]),
+            default="parallel",
+            help="The mode to process occurrences in. Batched-parallel uses batched in parallel to guarantee messages are processed in order per group, parallel uses multi-processing.",
+        )
+    )
+    return options
```

Comment:
personally I'd write:

```python
return [
    *multiprocessing_options(...),
    click.Option(
        ...
]
```

just a little more difficult to get wrong and the type checker can do better inference on the list type here (sometimes declaring a list and appending or otherwise can confuse the checker)

Reply:
Good idea